meson.build

```diff
@@ -48,6 +48,8 @@ endif
 conf.set_quoted('SHLEXT', shlext)
 conf.set_quoted('EXEEXT', exeext)
 
+p11_shlext = shlext
```

Comment:
I think this is not sufficient as Meson doesn't use `p11-kit-1.pc.in` as a template. Check how `p11_kit_pc_variables` is defined.

Reply:
fixed